src/OrchardCore/Orchard.Environment.Extensions/ExtensionManager.cs

```diff
@@ -90,6 +90,11 @@ public class ExtensionManager : IExtensionManager
 
         public IExtensionInfo GetExtension(string extensionId)
         {
+            if (string.IsNullOrEmpty(extensionId))
```

Comment:
I mean to throw an exception before this is an unexpected and unsupported state.